aws-predictions/src/main/java/com/amplifyframework/predictions/aws/AWSPredictionsPlugin.java

```diff
@@ -100,6 +105,16 @@ public String getPluginKey() {
         return AWS_PREDICTIONS_PLUGIN_KEY;
     }
 
+    /**
+     * Add version to liveness websocket user agent.
+     * @param type The type of version we're adding
+     * @param value The version
+     */
+    @InternalAmplifyApi
+    public static void addToUserAgent(AWSPredictionsMetadataType type, String value) {
```

Comment:
I'd rather just see user agent info added to the liveness calls, than hold onto the pairs.